bodhi/tests/server/services/test_updates.py

```diff
@@ -3088,6 +3088,9 @@ def test_disable_autopush_non_critical_update_with_negative_karma(self, publish,
         up.comment(self.db, u'Failed to work', author=u'ralph', karma=-1)
         up = self.db.query(Update).filter_by(title=resp.json['title']).one()
 
+        expected_comment = unicode(config.get('disable_automatic_push_to_stable'))
+        self.assertEquals(up.comments[2].text, expected_comment)
```

Comment:
One last thing to fix and I think we'll be good to go:

This test doesn't pass if ```development.ini``` doesn't have the new ```disable_automatic_push_to_stable``` setting. An easy way to fix it would be to put the default comment here in the call to ```config.get()``` too. It would be nice to allow development checkouts with existing development.ini files to keep working ☺

In the long run, I'd like to adjust ```bodhi.server.config``` so that all settings have sane defaults in a dictionary that is then overwritten by whatever is found in the config files. This way, all settings always have a default and we can always use ```config.get``` with one argument and can guarantee that it will be defined, regardless of what is in the ini files.